tests/postprocessors_test.rs

```diff
@@ -105,6 +107,34 @@ fn test_postprocessor_change_destination() {
     assert!(new_note_path.exists());
 }
 
+#[test]
+fn test_postprocessor_stateful_callback() {
```

Comment:
It might be a little hard to understand the value/purpose of this test when one is not familiar with the context of this PR. Let's add some explanation? Maybe something like:

```suggestion
// Ensure postprocessor type definition has proper lifetimes to allow state (here: `parents`)
// to be passed in. Otherwise, this fails with an error like:
//     error[E0597]: `parents` does not live long enough
//     cast requires that `parents` is borrowed for `'static`
#[test]
fn test_postprocessor_stateful_callback() {
```

Reply:
Thanks! Good idea, updated.

Reply:
Perfect! Thank you